test/src/e2e_vm_tests/test_programs/should_fail/main_should_not_have_args/test.toml

```diff
@@ -0,0 +1,4 @@
+category = "fail"
+
+# check: main(42)
+# nextln: $()Function "main" can't have any arguments. 
```

Comment:
Isn't the error message `arguments to "main()" are not yet supported.`?  That's what's in `error.rs` above...